### PR TITLE
Add recipe for platypus-opt

### PR DESCRIPTION
**Pull request by jetuk**

### CODE REVIEW

**Comment by snorfalorpagus** [comment]
@conda-forge-linter relint

**Comment by isuruf** [comment]
@conda-forge-linter relint

**Comment by jetuk** [comment]
@conda-forge-linter relint

**Comment by jetuk** [comment]
@isuruf is there anything I can do to move this long?

File: recipes/platypus-opt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  version: '{{ version }}'
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz  
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build_number }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (mariusvniekerk) [inline recipes/platypus-opt/meta.yaml]: Can you switch this to the new install form

```python -m pip install --no-deps --ignore-installed .```

File: recipes/platypus-opt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+requirements:
+  host:
+    - python
+    - setuptools
```
Comment (mariusvniekerk) [inline recipes/platypus-opt/meta.yaml]: remove `setuptools` and add `pip`

**Pull request merged**